src/components/ChildNodeViews.tsx

```diff
@@ -110,31 +121,34 @@ const ChildView = memo(function ChildView({
   getInnerPos,
 }: {
   child: Child;
-  getInnerPos: MutableRefObject<() => number>;
+  getInnerPos: () => number;
 }) {
   const { view } = useContext(EditorContext);
-  const getChildPos = useRef(() => getInnerPos.current() + child.offset);
-  getChildPos.current = () => getInnerPos.current() + child.offset;
+
+  const childRef = useRef<Child>(child);
+  childRef.current = child;
```

Comment:
I don't have an opinion on this, but I am curious: Why ref the whole child and not just the offset?

Reply:
No real reason at all. That's just how it flowed out as I did it, I guess.